README.md

```diff
@@ -1,6 +1,7 @@
 Git for Windows
 ===============
 
+[![Build status](https://git-for-windows.visualstudio.com/git/_apis/build/status/12?branch=master)](https://git-for-windows.visualstudio.com/git/_build/latest?definitionId=12&branch=master)
```

Comment:
Can the `Build status` alt text here and on the next line be used to differentiate these builds? It'd be cool if the image itself could embed the build name, but until then some sort of difference will help here without having to click into both...

Reply:
Good call. Can you see whether my new proposal addresses your wish?

Reply:
You still need to click on both if you are on mobile and can not hover over the buttons, but I guess that’s just a small tadeoff.